<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Philatelie</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frameFilter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBoxFilter">
          <item>
           <property name="text">
            <string>Year</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Group</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Category</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Designer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Engraver</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidgetFilter"/>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frameDisplay">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>horizontalSpacer</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>frameDisplay</zorder>
   <zorder>frameFilter</zorder>
   <zorder>splitter</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Philatelie"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCreate_new_stamp"/>
    <addaction name="actionSearch_stamp_in_collection"/>
    <addaction name="actionSearch_stamp_on_the_web"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCreate_new_stamp"/>
   <addaction name="actionSearch_stamp_in_collection"/>
   <addaction name="actionSearch_stamp_on_the_web"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/actions/document-open.png</normaloff>:/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_Philatelie">
   <property name="text">
    <string>About Philatelie</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/actions/document-save.png</normaloff>:/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionCreate_new_stamp">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/actions/add.png</normaloff>:/actions/add.png</iconset>
   </property>
   <property name="text">
    <string>Create new stamp</string>
   </property>
  </action>
  <action name="actionSearch_stamp_on_the_web">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/actions/applications-internet.png</normaloff>:/actions/applications-internet.png</iconset>
   </property>
   <property name="text">
    <string>Search stamp on the web</string>
   </property>
  </action>
  <action name="actionSearch_stamp_in_collection">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/actions/search.png</normaloff>:/actions/search.png</iconset>
   </property>
   <property name="text">
    <string>Search stamp in collection</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
